src/styles/utils/_extends.scss

```diff
@@ -1,4 +1,58 @@
-%h1 {
-  font-family: "Roboto", sans-serif;
-  font-weight: 400;
+%section-title {
+  font-family: $ff-alternate;
+    font-size: $fs-sm;
+    font-weight: $fw-bold;
+    text-transform: uppercase;
+    letter-spacing: 3px;
+    color: $clr-title-light;
+    line-height: $lh-md;
+
+    margin-bottom: 16px;
+}
+
+%title {
+    font-family: $ff-alternate;
+    font-size: $fs-2xl;
+    font-weight: $fw-normal;
+    color: $clr-title;
+    line-height: $lh-sm;
+}
+
+%sr-only {
+  border: 0 !important;
+  clip: rect(1px, 1px, 1px, 1px) !important;
+  -webkit-clip-path: inset(50%) !important;
+    clip-path: inset(50%) !important;
+  height: 1px !important;
+  margin: -1px !important;
+  overflow: hidden !important;
+  padding: 0 !important;
+  position: absolute !important;
+  width: 1px !important;
+  white-space: nowrap !important;
+}
+
+%btn {
+  display: inline-block;
+
+  background-color: $clr-accent;
+
+  box-sizing: border-box;
+  border: 4px solid $clr-accent;
+  border-radius: 8px;
+
+  color: $clr-natural;
+  text-align: center;
+
+  cursor: pointer;
+
+  transition:
+      color 0.2s cubic-bezier(.91,-0.01,1,.45),
+      background-color 0.2s cubic-bezier(.91,-0.01,1,.45),
+      box-shadow 0.2s cubic-bezier(.91,-0.01,1,.45);
+  &:hover {
```

Comment:
Hover effect in design is a bit different.  Background color changes to #144ED5 only.

![image](https://user-images.githubusercontent.com/65460971/203273411-de080637-2b03-44d2-8736-a023e41ae41c.png)


![image](https://user-images.githubusercontent.com/65460971/203273145-b29b53a0-a457-442a-ad63-8761c6069ae9.png)